cmd/osm-controller/osm-controller.go

```diff
@@ -98,6 +110,15 @@ func init() {
 
 	// feature flags
 	flags.BoolVar(&optionalFeatures.Backpressure, "enable-backpressure-experimental", false, "Enable experimental backpressure feature")
+
+	// k8s controller manager options
+	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+
+	_ = clientgoscheme.AddToScheme(scheme)
+	_ = v1beta1.AddToScheme(scheme)
+	// +kubebuilder:scaffold:scheme
```

Comment:
Could you describe as code comments what controller manager is, what metrics address means here, and so on.

Reply:
added some comments